src/stackwalk.c

```diff
@@ -1051,6 +1051,19 @@ void jl_rec_backtrace(jl_task_t *t)
     (void)mctx;
     (void)c;
   #endif
+ #elif defined(_OS_FREEBSD_) && defined(_CPU_X86_64_)
+    sigjmp_buf *mctx = &t->ctx.ctx.uc_mcontext;
+    mcontext_t *mc = &c.uc_mcontext;
+    // https://github.com/freebsd/freebsd-src/blob/releng/13.1/lib/libc/amd64/gen/_setjmp.S
+    mc->mc_rdx = ((long*)mctx)[0];
```

Comment:
```suggestion
    mc->mc_rip = ((long*)mctx)[0];
```
(rdx is just a temporary register in the assembly)

Reply:
Sorry, this is probably a dumb question but how did you determine that this should be `rip`?

Also, in case it's useful (though I'm guessing it isn't), [here](https://github.com/freebsd/freebsd-src/blob/releng/13.1/sys/x86/include/ucontext.h#L106-L164) is the FreeBSD definition of `mcontext_t` for x86-64.

Reply:
the assembly there (and expectations from having read/written this code often) loads it into rdx
`0(%rsp),%rdx		/* retval */`

fwiw, the Win64 code is based upon this struct's layout too, so they should end up being pretty similar